pep-0567.rst

```diff
@@ -573,25 +573,30 @@ C API
       if (PyContext_Exit(old_ctx)) goto error;
 
 
-Design Considerations
-=====================
+Rejected Ideas
+==============
+
+Replication of threading.local() interface
+------------------------------------------
+
+Please refer to :pep:`550` where this topic is covered in detail: [2]_.
 
-Why contextvars.Token and not ContextVar.unset()?
--------------------------------------------------
+
+Replace Token with ContextVar.unset()
```

Comment:
I'd like this title to use "Replacing" instead of "Replace". Ditto for a few other titles in the Rejected Ideas section (because out of context "replace xxx" sounds confusingly as if it's mandated rather than rejected).